### ISSUE TITLE
clients.frogclient.FrogClient assumes no FoLiA output

### ISSUE THREAD

**Issue by asharkinasuit** [proycon/pynlpl#38]
Title says it all, the FrogClient assumes the Frog server responds with a column format. When the server is running with -X, the output from FrogClient.process is [].

**Comment by proycon** [comment]
Indeed, the FrogClient is fairly old and simple and uses the column output. If you want better integration and FoLiA support, you might want to take a look at https://github.com/proycon/python-frog instead.

**Comment by asharkinasuit** [comment]
I know there is python-frog, but I'm using Visual Studio with Python on Windows... I have LaMachine installed in the WSL and while imperfect, it's a working solution to just run a Frog server in there. I'll see if I can use FoLiA to reshape the column output where needed.

**Comment by asharkinasuit** [comment]
Also, the constructor doesn't do anything with the `ner` parameter, and the `process` method doesn't fill in the `parse1` and `parse2` variables when `returnall == True`.

**Comment by proycon** [comment]
Okay, I see where you're coming from yeah. Adding FoLiA support may indeed be an interesting solution then, it should be fairly easy (at some overhead performance cost) so I'll take a look.

> Also, the constructor doesn't do anything with the ner parameter, and the process method doesn't fill in the parse1 and parse2 variables when returnall == True.

Hmm, indeed, well spotted.. looks like that was never properly finished and it seems nobody fell over it until now. I'll fix it
